invisible_cities/detsim/sensor_utils_test.py

```diff
@@ -97,3 +98,20 @@ def test_pmt_and_sipm_bin_width(full_sim_file):
     pmt_binwid, sipm_binwid = pmt_and_sipm_bin_width(file_in)
     assert pmt_binwid  == expected_pmtwid
     assert sipm_binwid == expected_sipmwid
+
+
+def test_create_timestamp_greater_with_greater_arguments():
```

Comment:
I'd maybe change `arguments` to `eventnumber` both in the name of the test and the docstring